example/metadata/bin/client.dart

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) 2017, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:metadata/src/client.dart';
+
+main(List<String> args) async {
+  await new Client().main(args);
```

Comment:
This looks nice, but you could also just mark this function non-async (drop the `async` modifier) and return the result of calling `new Client().main(args)`, right?